src/components/AssetIcon.tsx

```diff
@@ -559,7 +560,6 @@ const assetIcons = {
   TWT: '/currencies/twt.png',
   UFT: '/currencies/uft.png',
   UMA: '/currencies/uma.png',
-  UNAVAILABLE: '/currencies/unavailable.png',
```

Comment:
is this intended?

Reply:
Yeah, it was probably the linter that deleted this key.

Reply:
I deleted because it's from my script not handling error case. is this used as a placeholder though?